specification/orbital/resource-manager/Microsoft.Orbital/stable/2022-11-01/orbital.json

```diff
@@ -0,0 +1,2264 @@
+      ],
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Channel name."
+        },
+        "centerFrequencyMHz": {
+          "type": "number",
+          "description": "Center Frequency in MHz."
+        },
+        "bandwidthMHz": {
+          "type": "number",
+          "description": "Bandwidth in MHz."
+        },
+        "endPoint": {
+          "$ref": "#/definitions/EndPoint"
+        },
+        "modulationConfiguration": {
+          "type": "string",
+          "description": "Copy of the modem configuration file such as Kratos QRadio. Only valid for uplink directions. If provided, the modem connects to the customer endpoint and accepts commands from the customer instead of a VITA.49 stream."
+        },
+        "demodulationConfiguration": {
+          "type": "string",
+          "description": "Copy of the modem configuration file such as Kratos QRadio or Kratos QuantumRx. Only valid for downlink directions. If provided, the modem connects to the customer endpoint and sends demodulated data instead of a VITA.49 stream."
+        },
+        "encodingConfiguration": {
+          "type": "string",
+          "description": "Currently unused."
+        },
+        "decodingConfiguration": {
+          "type": "string",
+          "description": "Currently unused."
+        }
+      },
+      "description": "Contact Profile Link Channel."
+    },
+    "ContactProfileLink": {
+      "type": "object",
+      "required": [
+        "name",
+        "polarization",
+        "direction",
+        "channels"
+      ],
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Link name."
+        },
+        "polarization": {
+          "type": "string",
+          "description": "Polarization. e.g. (RHCP, LHCP).",
+          "enum": [
+            "RHCP",
+            "LHCP",
+            "linearVertical",
+            "linearHorizontal"
+          ],
+          "x-ms-enum": {
+            "name": "polarization",
+            "modelAsString": true
+          }
+        },
+        "direction": {
+          "type": "string",
+          "description": "Direction (uplink or downlink).",
+          "enum": [
+            "uplink",
+            "downlink"
+          ],
+          "x-ms-enum": {
+            "name": "direction",
+            "modelAsString": true
+          }
+        },
+        "gainOverTemperature": {
+          "type": "number",
+          "description": "Gain To Noise Temperature in db/K. It is the required G/T by the customer. Not used yet."
+        },
+        "eirpdBW": {
+          "type": "number",
+          "description": "Effective Isotropic Radiated Power (EIRP) in dBW. It is the required EIRP by the customer. Not used yet."
+        },
+        "channels": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ContactProfileLinkChannel"
+          },
+          "x-ms-identifiers": [
```

Comment:
ContactProfileLinkChannel has a `name` property. Shouldn't that be the x-ms-identifiers?